src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <>
+    <div className="movies">
+      {movies.map(movi => (
+        <div key={movi.title}>
+          <MovieCard movie={movi} />
+        </div>
```

Comment:
```suggestion
        <MovieCard movie={movi} key={movi.title} />
```
looks like unnecessary div